src/bip44.c

```diff
@@ -258,37 +300,8 @@ bip44_path_type_t bip44_classifyPath(const bip44_path_t* pathSpec)
 		}
```

Comment:
It perhaps makes sense to extract all 1853-related stuff into a separate function, e.g. `bip44_classifyPathPoolColdKey`.